scripts/s2i-generate.sh

```diff
@@ -51,6 +53,7 @@ fi
 phase "Generating the Containerfile for S2I builder image '${S2I_BUILDER_IMAGE}'"
 s2i --loglevel "${S2I_LOGLEVEL}" \
     build "${PARAMS_SUBDIRECTORY}" "${S2I_BUILDER_IMAGE}" \
+        --image-scripts-url "${IMAGE_SCRIPTS_URL}" \
```

Comment:
Also here, please use `PARAMS_IMAGE_SCRIPTS_URL` variable.

Reply:
I have updated the PR as per the review feedback.